src/chains/celo/parsers.ts

```diff
@@ -121,3 +131,74 @@ function parseTransactionCIP42(
 
   return transaction as TransactionSerializableCIP42
 }
+
+function parseTransactionCIP64(
+  serializedTransaction: TransactionSerializedCIP64,
+): TransactionSerializableCIP64 {
+  const transactionArray = toTransactionArray(serializedTransaction)
+
+  const [
+    chainId,
+    nonce,
+    maxPriorityFeePerGas,
+    maxFeePerGas,
+    gas,
+    feeCurrency,
+    to,
+    value,
+    data,
+    accessList,
+    v,
+    r,
+    s,
```

Comment:
the order here i think should have feeCurrency after access list

Reply:
oops. thanks!